server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/BallotManifestUpload.java

```diff
@@ -93,7 +95,7 @@ private UploadedFile attemptFilePersistence(final File the_file,
                                               final String the_hash) {
     UploadedFile result = null;
     
-    if (Persistence.isEnabled()) {
+    if (Persistence.hasDB()) {
```

Comment:
This actually made me notice that I had failed to clean up `UploadedFile`, so thanks for that. Though I'm not sure my fix will make you thrilled. :)